astropy/modeling/rotations.py

```diff
@@ -233,11 +232,11 @@ def __init__(self, phi, theta, psi, axes_order, **kwargs):
         if len(axes_order) != 3:
             raise TypeError(
                 "Expected axes_order to be a character sequence of length 3, "
-                "got {}".format(axes_order))
+                f"got {axes_order}")
         unrecognized = set(axes_order).difference(self.axes)
         if unrecognized:
-            raise ValueError("Unrecognized axis label {}; "
-                             "should be one of {} ".format(unrecognized, self.axes))
+            raise ValueError(f"Unrecognized axis label {unrecognized}; "
+                             f"should be one of {self.axes} ")
```

Comment:
Is the trailing whitespace necessary?
```suggestion
                             f"should be one of {self.axes}")
```